tests/fixtures/updated/LibWithInit/test_res_lib_dir.resource

```diff
@@ -0,0 +1,14 @@
+*** Settings ***
```

Comment:
it looks identical to the one from initial fixture directory. Is it necessary?

Reply:
Deleted